hw/sensor/src/sensor.c

```diff
@@ -76,15 +76,15 @@ static struct os_event sensor_interrupt_event = {
     .ev_cb = sensor_interrupt_ev_cb,
 };
 
-static struct os_event sensor_notify_event = {
-    .ev_cb = sensor_notify_ev_cb,
-};
-
-  /** OS event - for doing a sensor read */
+/** OS event - for doing a sensor read */
 static struct os_event sensor_read_event = {
     .ev_cb = sensor_read_ev_cb,
 };
 
+static struct os_mempool sensor_notify_evt_pool;
+static uint8_t sensor_notify_evt_area[OS_MEMPOOL_BYTES(MYNEWT_VAL(SENSOR_NOTIF_EVENTS_MAX),
+      sizeof(struct os_event))];
```

Comment:
Shouldn't this be `sizeof (struct sensor_notify_os_ev)`?

Reply:
yup, good catch. Thanks.